VATSpy.dat

```diff
@@ -17907,8 +17906,8 @@ ENBD|Bodo ACC - Polaris||ENBD
 ENBD-S|Bodo South ACC - Polaris|ENBD_S|ENBD-S
 ENOB|Bodo Oceanic||ENOB
 ENOR|Polaris||ENOR
+ENRC|Bodo RTC - Polaris||ENOR

```

Comment:
Mind putting this one in alphabetical order, pwease? 🙃

Reply:
While I highly disagree with it, I have done it.

Reply:
Many thanks! I do understand your concerns, but I’d rather keep consistency across the whole file.